internal/jira/jira_boards.go

```diff
@@ -39,3 +40,16 @@ func (api *httpApi) GetBoardConfiguration(boardId int) (*BoardConfiguration, err
 	}
 	return &result, nil
 }
+
+func (api *httpApi) GetFilter(filterId string) (*Filter, error) {
```

Comment:
the workflow goes as follow:

-> select project
-> show boards for that project

I don't see how to work with that `GetFilter(filterId string)` method. How to get appropriate filter id upfront?

Reply:
ah okay I see it now - it's in goto.go